src/main/java/com/redhat/devtools/lsp4ij/LanguageServerManager.java

```diff
@@ -0,0 +1,123 @@
+/*******************************************************************************
+ * Copyright (c) 2024 Red Hat, Inc.
+ * Distributed under license by Red Hat, Inc. All rights reserved.
+ * This program is made available under the terms of the
+ * Eclipse Public License v2.0 which accompanies this distribution,
+ * and is available at https://www.eclipse.org/legal/epl-v20.html
+ *
+ * Contributors:
+ * Red Hat, Inc. - initial API and implementation
+ ******************************************************************************/
+package com.redhat.devtools.lsp4ij;
+
+import com.intellij.openapi.project.Project;
+import com.redhat.devtools.lsp4ij.server.definition.LanguageServerDefinition;
+import org.jetbrains.annotations.NotNull;
+
+public class LanguageServerManager {
+
+    private final Project project;
+
+    public static class StartOptions {
+
+        private boolean forceStart;
+
+        public boolean isForceStart() {
+            return forceStart;
+        }
+
+        public void setForceStart(boolean forceStart) {
+            this.forceStart = forceStart;
+        }
+    }
+
+    public static class StopOptions {
+
+        private boolean stopAndDisable;
+
+         public StopOptions() {
+             setStopAndDisable(true);
+         }
+
+        public boolean isStopAndDisable() {
+            return stopAndDisable;
+        }
+
+        public void setStopAndDisable(boolean stopAndDisable) {
+            this.stopAndDisable = stopAndDisable;
+        }
+    }
+
+    public static LanguageServerManager getInstance(@NotNull Project project) {
+        return project.getService(LanguageServerManager.class);
+    }
+
+    private LanguageServerManager(Project project) {
+        this.project = project;
+    }
+
+    public void start(@NotNull String languageSeverId,
+                      @NotNull StartOptions options) {
+        var serverDefinition = LanguageServersRegistry.getInstance().getServerDefinition(languageSeverId);
+        if (serverDefinition == null) {
+            return;
+        }
+        start(serverDefinition, options);
+    }
+
+    public ServerStatus getServerStatus(String languageSeverId) {
+        var serverDefinition = LanguageServersRegistry.getInstance().getServerDefinition(languageSeverId);
+        if (serverDefinition == null) {
+            return null;
+        }
+        for (var ls : LanguageServiceAccessor.getInstance(project).getStartedServers()) {
+            if (serverDefinition == ls.getServerDefinition()) {
+                return ls.getServerStatus();
+            }
+        }
+            return ServerStatus.none;
+    }
+
+    public void start(@NotNull LanguageServerDefinition serverDefinition,
+                      @NotNull StartOptions options) {
+        boolean started = false;
+        for (var ls : LanguageServiceAccessor.getInstance(project).getStartedServers()) {
+            if (serverDefinition == ls.getServerDefinition()) {

```

Comment:
Use equals()

Reply:
Why we should do that since language server definition are created when LSP4IJ plugin is started?

Reply:
equals is 100% safe, just good habit

Reply:
fixed